docs/administration-guide/gaffer-config/config.md

```diff
@@ -43,53 +46,70 @@ that suites a stand alone deployment e.g. docker consisting of the following fil
     8. This file controls which containers will be started up and the configuration
     of them to ensure correct ports and files are available.
 
-All the files in the `config/accumulo/` and `config/hdfs/` directories will be copied directly from
-the two locations in the Gaffer docker repo,
-[here](https://github.com/gchq/gaffer-docker/tree/develop/docker/accumulo/conf-2.0.1) and
-[here](https://github.com/gchq/gaffer-docker/tree/develop/docker/hdfs/conf). The configuration of
-these are out of scope of this example but are covered in other sections of the documentation. The
-main focus of this guide will be on the configuration files under the `config/gaffer/` directory.
-
 ## Configuration Files
 
-### Application Properties
+### application.properties
 
-This is probably the simplest configuration file in the Gaffer deployment. In general it borrows a
-concept from [Spring Boot](https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html)
-to allow any properties related to Gaffer. In the example that follows
-we use it to set the file location properties of where the other config files are (inside the
-container).
+Within the application.properties file we set the file location properties of
+where the other config files are. In general it borrows a concept from [Spring
+Boot](https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html)
+to allow any properties related to Gaffer.
 
 ```properties title="application.properties"
 gaffer.schemas=/gaffer/schema
 gaffer.storeProperties=/gaffer/store/store.properties
 gaffer.graph.config=/gaffer/graph/graphConfig.json
 ```
 
-### Graph Configuration
+### graphConfig.json
 
-The graph configuration file is a JSON file that configures few bits of the Gaffer graph. Primarily
-it is used to set the name and description along with any additional hooks to run before an operation
-chain e.g. to impose limits on max results etc. For the example, as it is a very basic graph we just
-set the name and short description.
+Within the graphCongfig.json file you can set multiple properties. Here you define the `graphId` and `description` of the graph. You can also define any additonal `hooks` to run when operations are perfomed, a `view` you want to be applied when operations are perfomed and any addtional libraries with the `library` property.
```

Comment:
Maybe add a link to the graph metadata page as it goes into more detail about this file?

Reply:
Removed the graph metadata page as it contained less info than the config page. Added in the necessary missing parts into the config page.